Riot/Assets/de.lproj/Vector.strings

```diff
@@ -155,6 +155,7 @@
 "room_message_placeholder" = "Nachricht senden (unverschlüsselt)…";
 "encrypted_room_message_placeholder" = "Verschlüsselte Nachricht…";
 "room_message_short_placeholder" = "Sende eine Nachricht…";
+"room_first_message_placeholder" = "Sende deine erste Nachricht…";
```

Comment:
only the English version is expected in case of a new string (in order to not break merge from Weblate)